src/color/p5.Color.js

```diff
@@ -76,8 +76,18 @@ p5.Color = function(pInst, vals) {
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * createCanvas(200, 100);
+ * let myColor;
```

Comment:
could this line be combined with line 84?

Reply:
Sure, I will make the changes.
Thanks for the feedback.